src/main.rs

```diff
@@ -89,23 +90,59 @@ impl FromStr for Filter {
     }
 }
 
-#[derive(StructOpt)]
-#[structopt(
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+struct LevelFilterArg(LevelFilter);
+
+impl clap::ValueEnum for LevelFilterArg {
```

Comment:
why not using its `#derive` ?

Reply:
Ah yes, it is in the wrong place, probably I could update https://github.com/lu-zero/arg_enum_proc_macro

Reply:
The derive wouldn't work because it's a newtype wrapper. Not sure if `arg_enum_proc_macro` could support that without having to pass the all the inner type's variants in some way